README.md

```diff
@@ -11,319 +11,26 @@ focus on sparse solution of linear systems. It is implemented using modern C++
-                first, and using the fastest format determined by that
-                benchmark. The maximum number of iterations for the iterative
-                solvers is set to 10,000 and the requested residual reduction
-                factor to 1e-6.
-    *   `preconditioner` - Runs the preconditioner benchmarks on artificially
-                generated block-diagonal matrices.
-*   `DRY_RUN={true, false}` - If set to `true`, prepares the system for the
-    benchmark runs (downloads the collections, creates the result structure,
-    etc.) and outputs the list of commands that would normally be run, but does
-    not run the benchmarks themselves. Default is `false`.
-*   `EXECUTOR={reference,cuda,omp}` - The executor used for running the
-    benchmarks. Default is `cuda`.
-*   `SEGMENTS=<N>` - Splits the benchmark suite into `<N>` segments. This option
-    is useful for running the benchmarks on an HPC system with a batch
-    scheduler, as it enables partitioning of the benchmark suite and running it
-    concurrently on multiple nodes of the system. If specified, `SEGMENT_ID`
-    also has to be set. Default is `1`.
-*   `SEGMENT_ID=<I>` - used in combination with the `SEGMENTS` variable. `<I>`
-    should be an integer between 1 and `<N>`. If specified, only the `<I>`-th
-    segment of the benchmark suite will be run. Default is `1`.
-*   `SYSTEM_NAME=<name>` - the name of the system where the benchmarks are being
-    run. This option only changes the directory where the benchmark results are
-    stored. It can be used to avoid overwriting the benchmarks if multiple
-    systems share the same filesystem, or when copying the results between
-    systems. Default is `unknown`.
-
-Once `make benchmark` completes, the results can be found in
-`<Ginkgo build directory>/benchmark/results/${SYSTEM_NAME}/`. The files are
-written in the JSON format, and can be analyzed using any of the data
-analysis tools that support JSON. Alternatively, they can be uploaded to an
-online repository, and analyzed using Ginkgo's free web tool
-[Ginkgo Performance Explorer (GPE)](https://ginkgo-project.github.io/gpe/).
-(Make sure to change the "Performance data URL" to your repository if using
-GPE.)
-
-### Installing Ginkgo
-
-To install Ginkgo into the specified folder, execute the following command in
-the build folder
-
-```sh
-make install
-```
-
-If the installation prefix (see `CMAKE_INSTALL_PREFIX`) is not writable for your
-user, e.g. when installing Ginkgo system-wide, it might be necessary to prefix
-the call with `sudo`.
+A unique feature of Ginkgo is the ability to run benchmarks and view your results
+with the help of the [Ginkgo Performance Explorer (GPE)](https://ginkgo-project.github.io/gpe/). 
 
-After the installation, CMake can find ginkgo with `find_package(Ginkgo)`.
-An example can be found in the [`test_install`](test_install/CMakeLists.txt).
+More details about this can be found in the [documentation]()
 
 ### Licensing
```

Comment:
In the ABOUT-LICENSING.md you need to update the link at the very end (linking to the installation part of the readme).